charts/opentelemetry-kube-stack/templates/_helpers.tpl

```diff
@@ -286,3 +286,22 @@ Helpers for prometheus servicemonitors
   {{- $userValue := index . 3 -}}
   {{- include "opentelemetry-kube-stack.kubeVersionDefaultValue" (list $values ">= 1.23-0" $insecure $secure $userValue) -}}
 {{- end -}}
+
+{{/* Sets default scrape limits for servicemonitor */}}
+{{- define "servicemonitor.scrapeLimits" -}}
```

Comment:
We should update the name to match our naming scheme: `opentelemetry-kube-stack.servicemonitor.scrapeLimits`